src/controller/orderController.js

```diff
@@ -37,14 +37,14 @@ exports.checkOut = async (req, res) => {
 
     // Creating record in order table for history
     await orderModel.create({
-      userId: req.params.userId,
+      userid: req.params.userid,
```

Comment:
why changed and used `userid` instead of `userId`

Reply:
resolved